spk/python3/src/requirements.txt

```diff
@@ -9,7 +9,7 @@
 
 ## Cross-compiled via spksrc.wheel.mk
 markupsafe==1.1.1
-psutil==5.7.0
+#psutil==5.7.0
```

Comment:
Got my old DS112j up and running again. I installed your build and it seems to work.
First thing I noticed is that we include all the requirements for `cryptograpghy` in the `requirements.txt` (few lines below), but that we don't actually include it in the `spk/python3`. So let's add that back in, just like we do for `spk/python`.

Reply:
@smaarn Me again 😄 I wanted to share some work I have been doing: 
So, while trying for hours to get `spk/sabnzbd` to run, it kept failing on `cryptography` not being available. Inspecting the `package.tgz` it turns out that the `package.tgz\lib\python3.7\site-packages` did not contain any of the `cryptography` files. Even though the build log shows they are being created.
After digging I found that is because the `PLIST` for `cross/cryptography` [is empty](https://github.com/SynoCommunity/spksrc/blob/master/cross/cryptography/PLIST). This is not a problem if it is part of `python3` package, because there the `PLIST` contains just, well, _everything_. 
So either we update the `PLIST` of `cross/cryptography` or we just include it in `pyton3`.

Reply:
Working on it while also fixing the ppc853x architecture build issues.

Reply:
> Working on it while also fixing the ppc853x architecture build issues.

I just picked this up and solved it by moving the wheel build of rdiff-backup from spk/rdiff-backup/Makefile to cross/rdiff-backup (including all dependencies except BUILD_DEPENDS)
My intent was to add a patch for ppx853x and hoped to solve this with cross/rdiff-backup. But luckily the build succeeded with this approach without a patch (no unknown compiler option -Wno-unused-result).